File: sdkwork-backend-sdk-go/types/plus_chat_options.go
package types


type PlusChatOptions struct {
	StreamUsage bool `json:"streamUsage"`
	Model string `json:"model"`
	FrequencyPenalty float64 `json:"frequency_penalty"`
	LogitBias map[string]int `json:"logit_bias"`
	Logprobs bool `json:"logprobs"`
	TopLogprobs int `json:"top_logprobs"`
	MaxTokens int `json:"max_tokens"`
	MaxCompletionTokens int `json:"max_completion_tokens"`
	N int `json:"n"`
	Modalities []string `json:"modalities"`
	Audio AudioParameters `json:"audio"`
	PresencePenalty float64 `json:"presence_penalty"`
	ResponseFormat ResponseFormat `json:"response_format"`
	StreamOptions StreamOptions `json:"stream_options"`
	Stream bool `json:"stream"`
	Seed int `json:"seed"`
	Stop []string `json:"stop"`
	Temperature float64 `json:"temperature"`
	TopP float64 `json:"top_p"`
	Tools []FunctionTool `json:"tools"`
	ToolChoice interface{} `json:"tool_choice"`
	User string `json:"user"`
	ParallelToolCalls bool `json:"parallel_tool_calls"`
	Store bool `json:"store"`
	Metadata map[string]string `json:"metadata"`
	ReasoningEffort string `json:"reasoning_effort"`
	Verbosity string `json:"verbosity"`
	WebSearchOptions WebSearchOptions `json:"web_search_options"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusChatOptions.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusChatOptions
    {
        public bool? StreamUsage { get; set; }
        public string? Model { get; set; }
        public double? FrequencyPenalty { get; set; }
        public Dictionary<string, int>? LogitBias { get; set; }
        public bool? Logprobs { get; set; }
        public int? TopLogprobs { get; set; }
        public int? MaxTokens { get; set; }
        public int? MaxCompletionTokens { get; set; }
        public int? N { get; set; }
        public List<string>? Modalities { get; set; }
        public AudioParameters? Audio { get; set; }
        public double? PresencePenalty { get; set; }
        public ResponseFormat? ResponseFormat { get; set; }
        public StreamOptions? StreamOptions { get; set; }
        public bool? Stream { get; set; }
        public int? Seed { get; set; }
        public List<string>? Stop { get; set; }
        public double? Temperature { get; set; }
        public double? TopP { get; set; }
        public List<FunctionTool>? Tools { get; set; }
        public object? ToolChoice { get; set; }
        public string? User { get; set; }
        public bool? ParallelToolCalls { get; set; }
        public bool? Store { get; set; }
        public Dictionary<string, string>? Metadata { get; set; }
        public string? ReasoningEffort { get; set; }
        public string? Verbosity { get; set; }
        public WebSearchOptions? WebSearchOptions { get; set; }
    }
}
